### DISCUSSION TITLE
Feature Suggest: Save the Current Comments

### DISCUSSION THREAD

**Discussion by Xovee** [openreview/openreview#248] [Ideas]
Hi, 

Currently I am writing rebuttal for my papers submitted to a conference. I found that I cannot save my current draft of the rebuttal, or any instances of the comments. Therefore I suggest a new feature that any users can save their draft when writing rebuttals/comments. For example, add a save button here. 

![image](https://github.com/user-attachments/assets/291b7846-d6ee-417e-9ab4-b32591c65ac8)

With such feature I no longer need to write draft in another Markdown platform/software and then have to paste them to the OpenReview page. 

Best, 
Xovee Xu

**Comment by carlosmondra** [comment]
If you use the same browser in the same device, your progress is saved automatically in the localstorage of the browser. Does this work for you?

Implementing saving a draft in the database is in our plan, but it's not a priority at the moment.

**Reply by Xovee** [reply]
Works for me. Did not notice this local-saving feature before. Maybe you can add some hints in the placeholder of the text input box or above it. Just a thought : )

OpenReview is fantastic. Thank you for all your hard work on it, and hope everything is going well. 

Have a great weekend!
